src/test/groovy/org/prebid/server/functional/model/request/auction/StoredAuctionResponse.groovy

```diff
@@ -1,9 +1,11 @@
 package org.prebid.server.functional.model.request.auction
 
 import groovy.transform.ToString
+import org.prebid.server.functional.model.response.auction.SeatBid
 
 @ToString(includeNames = true, ignoreNulls = true)
 class StoredAuctionResponse {
 
     String id
+    List<SeatBid> seatbid
```

Comment:
@marki1an there was changes in naming and logic

https://github.com/prebid/prebid-server/issues/2518#issuecomment-2176749147